provider/auth.md

```diff
@@ -34,6 +34,6 @@ If an MDS `provider` implements this auth scheme, it **MAY** choose to specify t
 
 ## Endpoint Authentication Requirements  
 
-The `/trips` and `/status_changes` endpoints may or may not require authentication, it is up to the implementing provider to decide if they want historical information available with or without authentication. 
+The `/trips` and `/status_changes` endpoints must require authentication.
```

Comment:
If we're changing this, may as well update to be more general now that we have `/events` and `/vehicles` endpoints as well.

Reply:
done